src/apps/collectionsUi/viewModules/home/Async.js

```diff
@@ -1,7 +1,8 @@
 import { createAsyncView } from 'coreModules/bootstrap/higherOrderComponents'
+import { name } from './index'
```

Comment:
Its not valid to import in from index. then the async aspect makes no sense. If you dont want duplication add a constants file